filament/src/Renderer.cpp

```diff
@@ -366,12 +366,13 @@ void FRenderer::renderJob(ArenaScope& arena, FView& view) {
                 .keepOverrideEnd = keepOverrideEndFlags
         }, viewRenderTarget);
 
-    auto importTexture = [&](const FTexture* texture, const char* name) {
+    auto importTexture = [&](const FTexture* texture, const char* name, uint8_t samples = 0) {
         FrameGraphTexture frameGraphTexture{ .handle = texture->getHwHandle() };
         return fg.import(name, {
                 .width = (uint32_t)texture->getWidth(0u),
                 .height = (uint32_t)texture->getHeight(0u),
-                .format = texture->getFormat()
+                .samples = samples,
```

Comment:
On a related note, why can't this be `(uint8_t)texture->getSampleCount()`?

Reply:
fixed in https://github.com/shapr3d/filament/pull/67/commits/917a91248e3bd5fdf063c55e3dcdbf473fc76d29